Source/JavaScriptCore/b3/testb3_1.cpp

```diff
@@ -936,6 +936,8 @@ int main(int argc, char** argv)
                 usage();
         } else if (!strcmp(argv[i], "-list"))
             config.mode = TestConfig::Mode::ListTests;
+        else if (!strcmp(argv[i], "-printir"))
+            g_dumpB3AfterGeneration = true;
```

Comment:
And if there is an option, this flag is not necessary. We can just pass an option via environment variable.